custom_components/tuya_local/devices/woods_milan_airconditioner.yaml

```diff
@@ -0,0 +1,69 @@
+name: Woods Milan Air Conditioner
+products:
+  - id: i5rv3ykmtfcagw1t
+    name: Wood's Milan 7K
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      persists: true
+      mapping:
+        - dps_val: false
+          value: "off"
+          icon: "mdi:hvac-off"
+        - dps_val: true
+          constraint: mode
+          conditions:
+            - dps_val: auto
+              icon: mdi:thermostat-auto
+              value: auto
```

Comment:
Normally "auto" maps to "heat_cool" in HA terminology - the mode that automatically switches between heat and cool according to whether abbient temperature is above or below the target temperature.  In HA terminology "auto" is reserved for a mode that follows a schedule or monitors activity, or something else smarter than the standard A/C modes..

Reply:
I believe this is correct as the device itself has a auto mode. Both in the dp and functionally on the device

Reply:
I am talking about the list of modes here https://developers.home-assistant.io/docs/core/entity/climate/#hvac-modes

95% of air conditioners have a mode labelled auto which corresponds to heat_cool in HA.  It is rare to find what HA calls auto, especially in a portable unit.  In the HA UI you will mostly only see icons anyway - the auto icon is a calendar, the heat_cool icon is a loop with arrows (mdi:sync).